src/pages/homeCollection/kitAssembly.js

```diff
@@ -162,667 +26,199 @@ const kitAssemblyTemplate = async (user, name, auth, route) => {
-            <td>${kitData[i].supplyKitId}</td>
-            <td>${kitData[i].specimenKitId}</td>
-            <td>${kitData[i].collectionCupId}</td>
-            <td>${kitData[i].collectionCardId}</td>
-        </tr>`;
-
-    // Update the last row number
-    lastRowNumber = i + 1;
-    // // If the current iteration is the last item and matches length of last row variable, add an extra row
-    if (lastRowNumber === kitData.length) {
-      extraRow = `
-        <tr class="new-row">      
-          <th scope="row">${lastRowNumber + 1}</th>
-          <td>
-            <input id="input-usps" class="input-field" autocomplete="off" name="input-usps" style="width:100%;text-overflow: ellipsis;" placeholder="9221690209813300440662" />
-            <label for ="input-usps" style="font-size:.8rem;">Ex. 9221690209813300440662</label>
-            <p id="input-usps-error-message" class="input-error-message"></p>
-          </td>
-          <td>
-            <input id="input-supply-kit" class="input-field" type="string" autocomplete="off" name="input-supply-kit" style="width:100%" placeholder="CON000007"/>
-            <label for ="input-supply-kit" style="font-size:.8rem;">Ex. CON000007</label>
-            <p id="input-supply-kit-error-message" class="input-error-message"></p>
-          </td>
-          <td>
-            <input id="input-specimen-kit" class="input-field" type="string" autocomplete="off" name="input-specimen-kit" style="width:100%" name="input-specimen-kit" placeholder="CON000007"/>
-            <label for ="input-specimen-kit" style="font-size:.8rem;">Ex. CON000007</label>
-            <p id="input-specimen-kit-error-message" class="input-error-message"></p>
-          </td>
-          <td>
-            <input id="input-collection-cup" class="input-field" type="string" autocomplete="off" style="width:100%;" placeholder="CXA123460 0007" name="input-collection-cup"/>
-            <label for ="input-collection-cup" style="font-size:.8rem;">Ex. CXA123460 0007
-            </label>
-            <p id="input-collection-cup-error-message"
-            class="input-error-message"></p>
-          </td>
-          <td>
-              <input id="input-collection-card" class="input-field" type="string" autocomplete="off" style="width:100%" placeholder="CXA123460 0007" name="input-collection-card"/>
-              <label for ="input-collection-card" style="font-size:.8rem;">Ex. CXA123460 0007
-              </label>
-              <p id="input-collection-card-error-message" class="input-error-message"></p>
-          </td>
-      </tr>
-      `;
-      tableRow += extraRow;
+const enableEnterKeystroke = () => {
+  document.getElementById("cardId")
+    .addEventListener("keyup", (event) => {
+    event.preventDefault();
+    if (event.key === 'Enter') {
+        document.getElementById("saveKit").click();
     }
+});
+}
 
-    tableBody.innerHTML = tableRow;
+const processAssembledKit = () => {
+  const a = document.getElementById('saveKit');
```

Comment:
We could rename `a` to `saveKitButton` or another variable name to be more descriptive.